client/pages/login.tsx

```diff
@@ -1,20 +1,51 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, useContext } from "react";
 import Layout from "../components/Layout";
 import Cookie from "js-cookie";
 import { parseCookies } from "../lib/parseCookies";
-import auth0 from "auth0-js";
+import createAuth0Client from "@auth0/auth0-spa-js";
+
 // Load AuthForm as an AMP page
 export const config = { amp: "hybrid" };
 
+export var auth0 = null;
+export const getAuth0Provider = async () => {
+  if (auth0 != null) {
+    return auth0;
+  }
+  auth0 = await createAuth0Client({
+    domain: "dev-l4sg3wav.auth0.com",
```

Comment:
prob wanna put these in `.env` so we can have a different auth0 system for prod

Reply:
Probably gonna have a followup PR that refactors a couple of usage things, since we'd want this auth0 object to be a app wide global singleton, and then also having it be .env configurable. Idk if you have a preference of just making those changes in this PR or having another one on top.

Reply:
cool down to just do that in another PR